modules/ROOT/pages/upgrade-cluster.adoc

```diff
@@ -191,7 +191,7 @@ sudo gravity plan --repair
 . On the controller node running the upgrade, run the failed phase manually:
 +
 ----
-sudo ./gravity plan execute --phase=< insert phase > --force --debug
+sudo gravity plan execute --phase=< insert phase > --force --debug
```

Comment:
same comment as above (which gravity binary to use in the case of upgrades)

Reply:
in this case the new version from the `installer` directory as that is the one running the upgrade